tests/bluetooth/host/mock_host/src/hci_core.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/zephyr.h>
+#include <zephyr/bluetooth/hci.h>
+#include <host/hci_core.h>
+
+static void init_work(struct k_work *work);
+
+struct bt_dev bt_dev = {
+	.init          = Z_WORK_INITIALIZER(init_work),
+#if defined(CONFIG_BT_DEVICE_APPEARANCE_DYNAMIC)
+	.appearance = CONFIG_BT_DEVICE_APPEARANCE,
+#endif
+};
```

Comment:
@jori-nordic meant that you don't need the initializer.

```suggestion
/* Mock */
struct bt_dev bt_dev;
```